animation_nodes/math/rotation_conversion.pyx

```diff
@@ -1,6 +1,11 @@
-from libc.math cimport atan2, sqrt, hypot, cos, sin
+cimport cython
+from libc.math cimport M_PI as PI, sqrt, hypot, abs, sin, cos, asin, acos, atan2, copysign
 from . matrix cimport normalizeMatrix_3x3_Part
 from . conversion cimport toPyEuler3
+from . vector cimport lengthVec3
+from . quaternion cimport setUnitQuaternion
+from mathutils import Vector, Matrix, Euler
+from .. math cimport quaternionNormalize_InPlace
```

Comment:
Order the imports according to the line length. Short at the top, long at the button. Break long lines if necessary, see other files to see how this can be done.